docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -329,6 +329,10 @@ Required when `actions` are used to send notifications.
 
 |version |Integer |The rule's version number. Defaults to `1`.
 
+|investigation_fields |String a| Specify additional highlighted fields for the alert:
```

Comment:
What do you think of:

```
Specify additional highlighted fields for select alert investigation flows:
```

Reply:
Are "select alert investigation flows" personalized investigation flows? Or are we saying that customized highlighted fields only select _a few_ investigation flows? If it's the former, I can modify the description to:

Specify highlighted fields for personalized alert investigation flows.

Reply:
That's a good question. With "select alert investigation flows" I was trying to reference the fact that these fields are used for more than just the alert details > highlighted fields section (like alert details > prevelance and exceptions). But maybe that's just more confusing. 

And ++ to adding that field description to the create rule docs. Thanks for catching that!

Reply:
Ok, cool. So just to wrap this up, do you think the description is good as is, or should I change it to: 
```suggestion
|investigation_fields |String a| Specify highlighted fields for personalized alert investigation flows:
```